R/LFMCMC.R

```diff
@@ -241,10 +241,10 @@ set_par_names <- function(lfmcmc, names) {
 #' @param names The model stats names
 #' @returns The lfmcmc model with the stats names added
 #' @export
-set_stats_names <- function(lfmcmc, names) {
+set_stat_names <- function(lfmcmc, names) {
```

Comment:
I see `set_stat_names` and down below `get_n_stats`. Let's use the `stats` and `params` too.